### DISCUSSION TITLE
mapProps源码中的observer是否是多余的

### DISCUSSION THREAD

**Discussion by wanghansong** [alibaba/formily#3200] [Q&A 2.x]
https://github.com/alibaba/formily/blob/91d648893468486e43267487f938c342852c6b24/packages/vue/src/shared/connect.ts#L39

上链接源码那一行，defineComponent就已经是生成了响应式组件，为啥要加一层observer

**Comment by yiyunwan** [comment]
并没有啊，那个是vue的响应式，这个是formily的响应式，不一样的
